sys/random/fortuna/fortuna.c

```diff
@@ -67,7 +67,10 @@ static int fortuna_generate_blocks(fortuna_state_t *state, uint8_t *out,
     }
 
     for (size_t i = 0; i < blocks; i++) {
-        aes_encrypt(&cipher, state->gen.counter.bytes, out + (i * 16));
+        res = aes_encrypt(&cipher, state->gen.counter.bytes, out + (i * 16));
+        if (res != 1) {
+            return res;
```

Comment:
I wrote this implementation some time ago. I think the return code should be negative, and I'm not sure about `res` being negative.

Reply:
Frme [the doc](https://doc.riot-os.org/aes_8h.html#aaca6a1300bb44f1513a28de5de8f9774):

>    1 on success 
>    A negative value if the cipher key cannot be expanded with the AES key schedule 

`fortuna_generate_blocks` is an internal function so it should be fine to return the negative value. *But* maybe an "internal" documentation of that function would be beneficial.